src/forms/FormStateApp.tsx

```diff
@@ -63,113 +74,75 @@ export function FormStateApp() {
     [],
   );
 
-  const sports = [
-    { id: undefined as any, name: "Undecided" },
-    { id: "s:1", name: "Football" },
-    { id: "s:2", name: "Soccer" },
-  ];
-
-  const colors: CheckboxGroupItemOption[] = [
-    { value: "c:1", label: "Blue" },
-    { value: "c:2", label: "Red" },
-    { value: "c:3", label: "Green" },
-  ];
-
-  const shapes = [
-    { id: "sh:1", name: "Triangle" },
-    { id: "sh:2", name: "Square" },
-    { id: "sh:3", name: "Circle" },
-  ];
-
-  const animals = [
-    { value: "a:1", label: "Cat" },
-    { value: "a:2", label: "Dog" },
-    { value: "a:3", label: "Fish" },
-    { value: "a:4", label: "Iguana" },
-    { value: "a:5", label: "Turtle" },
-  ];
-
-  const genres: NestedOption<HasIdAndName>[] = [
-    {
-      id: "g:1",
-      name: "Action",
-      children: [
-        {
-          id: "g:2",
-          name: "Action Adventure",
-          children: [{ id: "g:3", name: "Action Adventure Comedy" }],
-        },
-        { id: "g:4", name: "Action Comedy" },
-      ],
-    },
-    { id: "g:5", name: "Comedy", children: [{ id: "g:6", name: "Comedy Drama" }] },
-  ];
-
   return (
-    <Observer>
-      {() => (
```

Comment:
Removed the `Observer` wrapper since it was only used for the form buttons, which now use SubmitButton